package.json

```diff
@@ -40,7 +40,7 @@
     "aigle": "git+https://github.com/suguru03/aigle#8739846ba9d4cfc116e1546da1181c73564cae0b",
     "aware": "^0.3.1",
     "bignumber.js": "^9.0.0",
-    "dvf-utils": "git+https://github.com/DeversiFi/dvf-utils#refs/tags/v1.4.0",
+    "dvf-utils": "git+https://github.com/DeversiFi/dvf-utils#refs/tags/v1.4.3",
```

Comment:
@sjpbeale note that dvf-utils is now automatically published to [npm](https://www.npmjs.com/package/@rhino.fi/dvf-utils) in each commit to master (as long as the version is bumped), so you could simple do 

"@rhino.fi/dvf-utils": "1.4.3"

(after replacing all the require to use the scoped package name)